### DISCUSSION TITLE
In editor help for gdscript keywords

### DISCUSSION THREAD

**Discussion by aXu-AP** [godotengine/godot-proposals#4641] [Editor]
I very much like it that Godot ships with reference built-in and accessible inside the editor. However, for whatever reason I can't find description for keywords (and annotations in 4.0). From time to time I forget how to use `yield` properly or certain exports. For newcomers even things in template script like `extends` and `func` can cause questions. It would be helpful to be able ctrl+click those words as well or hit F1 and search for it. It's even more confusing since some keywords (highlighted in editor in red), as in primitive types (int, float, bool) do have their own page.

When I first started learning Godot, I had programming background. Concepts were familiar, but syntax not. Quick peek into reference would accelerate learning of gdscript for those already into programming.

As a clarification, I'm only talking about in-editor reference. Docs of course cover these topics in sufficient manner.

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/2361.
